<launch>
  <node name="drone_stateestimation" pkg="tum_ardrone" type="drone_stateestimation">
  </node>
  <node name="drone_autopilot" pkg="tum_ardrone" type="drone_autopilot">
  </node>
  <node name="drone_gui" pkg="tum_ardrone" type="drone_gui">
  </node>
  <node name="dronevideo_pub" pkg="tum_ardrone" type="dronevideo_pub">
  </node>
  <node name="dronevideo_sub" pkg="tum_ardrone" type="dronevideo_sub">
  </node>
</launch>
